pkg/network/ovn_kubernetes.go

```diff
@@ -118,6 +118,11 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	data := render.MakeRenderData()
 	data.Data["ReleaseVersion"] = os.Getenv("RELEASE_VERSION")
 	data.Data["OvnImage"] = os.Getenv("OVN_IMAGE")
+	data.Data["OvnNodeImage"] = os.Getenv("OVN_IMAGE")
+	data.Data["OvnMasterImage"] = os.Getenv("OVN_IMAGE")
+	if bootstrapResult.OVN.OVNKubernetesConfig.HyperShiftConfig.Enabled {
+		data.Data["OvnMasterImage"] = os.Getenv("OVN_CONTROL_PLANE_IMAGE")
```

Comment:
NIT: Move the image to `HyperShiftConfig`, something like: https://github.com/openshift/cluster-network-operator/blob/b817cd90ccc3528ca025d488bf0145d42831fa41/pkg/platform/hypershift.go#L35